atlasdb-client/src/main/java/com/palantir/atlasdb/logging/LoggingArgs.java

```diff
@@ -34,14 +34,25 @@
 import com.palantir.logsafe.Arg;
 import com.palantir.logsafe.SafeArg;
 import com.palantir.logsafe.UnsafeArg;
-import com.palantir.util.Pair;
 
 /**
  * Includes utilities for generating logging args that may be safe or unsafe, depending on table metadata.
  *
  * Always returns unsafe, until hydrated.
  */
 public final class LoggingArgs {
+    public static class SafeAndUnsafeTableReferences {
```

Comment:
Could use `@Value.Immutable` here.

Reply:
I thought about it... Didn't feel that I needed to, but I just added it anyway. Makes the code look better. :)